vagrant_provision.sh

```diff
@@ -52,9 +52,21 @@ cat > /home/vagrant/hadoop3/etc/hadoop/hdfs-site.xml <<EOF
     </property>
 </configuration>
 EOF
+
+# add custom config files to Hadoop
+cat /home/vagrant/rdfs/config/hdfs-site.xml > /home/vagrant/hadoop/etc/hadoop/hdfs-site.xml
+cat /home/vagrant/rdfs/config/core-site.xml > /home/vagrant/hadoop/etc/hadoop/core-site.xml
```

Comment:
can you remove the lines above this which touch the same files but get overwritten here?
`cat > /home/vagrant/hadoop3/etc/hadoop/core-site.xml <<EOF`
and
`cat > /home/vagrant/hadoop3/etc/hadoop/hdfs-site.xml <<EOF`